code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -47,8 +47,6 @@
 	for(var/mob/dead/observer/O in player_list)
 		if(check_observer(O))
 			to_chat(O, "<span class='boldnotice'>\A [src] has been activated. (<a href='?src=\ref[O];jump=\ref[src]'>Teleport</a> | <a href='?src=\ref[src];signup=\ref[O]'>Sign Up</a>)</span>")
-//			if(ROLE_POSIBRAIN in O.client.prefs.be_special) The Guardian implementation looks cleaner
```

Comment:
That's because I removed it, and for some reason didn't like the present implementation of "question".